### PR TITLE
Removes unnecessary Felix write permission as not required

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Removes unnecessary Felix write permission for some reason added to v3 and master docs but not present in v2.6 docs and not necessary. Raised by community, security implications.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: master/reference/advanced/etcd-rbac/calico-etcdv3-paths.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ component needs access to in etcd to function successfully.
 | /calico/ipam/v2/\*                        |   RW   |
 | /calico/resources/v3/projectcalico.org/\* |   RW   |
 
-## felix as a stand alone process
+## Felix as a stand alone process
 
 | Path                                      | Access |
 |-------------------------------------------|--------|
-| /calico/felix/v1/\*                       |   RW   |
+| /calico/felix/v1/\*                       |   R    |
```
Comment (caseydavenport) [inline master/reference/advanced/etcd-rbac/calico-etcdv3-paths.md]: I think this isn't quite right - `/calico/felix/v1/` is where Felix will write its status reports if enabled, so it does need that permission.

However, it doesn't need write access to `/calico/resources/v3`

Comment (emanic) [reply master/reference/advanced/etcd-rbac/calico-etcdv3-paths.md]: Oh OK, interesting. `/calico/resources/v3` is not mentioned in this table as being required, so I guess we do not need to make any change in that regard. 

Would you say then that this page is wrong https://docs.projectcalico.org/v2.6/reference/advanced/etcd-rbac/calico-etcdv2-paths and Felix actually needed `W` for `/calico/felix/v1/in v2.6?

Comment (caseydavenport) [reply master/reference/advanced/etcd-rbac/calico-etcdv3-paths.md]: > Oh OK, interesting. /calico/resources/v3 is not mentioned in this table as being required, so I guess we do not need to make any change in that regard.

Pretty sure it is, on the next line.

Comment (emanic) [reply master/reference/advanced/etcd-rbac/calico-etcdv3-paths.md]: Oh dang, oops. LOL

**Comment by emanic** [comment]
Thanks @fasaxc & @caseydavenport

**Pull request merged**